[2024-06-30T09:34:14.947+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Automate_cleaning.fetch_raw_data manual__2024-06-30T09:33:33.000476+00:00 [queued]>
[2024-06-30T09:34:15.039+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Automate_cleaning.fetch_raw_data manual__2024-06-30T09:33:33.000476+00:00 [queued]>
[2024-06-30T09:34:15.050+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-06-30T09:34:15.052+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-06-30T09:34:15.053+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-06-30T09:34:15.118+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): fetch_raw_data> on 2024-06-30 09:33:33.000476+00:00
[2024-06-30T09:34:15.153+0000] {standard_task_runner.py:52} INFO - Started process 247 to run task
[2024-06-30T09:34:15.171+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Automate_cleaning', 'fetch_raw_data', 'manual__2024-06-30T09:33:33.000476+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/automate_clean_DAG.py', '--cfg-path', '/tmp/tmpjiop7ry3', '--error-file', '/tmp/tmpjm0for2a']
[2024-06-30T09:34:15.184+0000] {standard_task_runner.py:80} INFO - Job 15: Subtask fetch_raw_data
[2024-06-30T09:34:15.614+0000] {task_command.py:371} INFO - Running <TaskInstance: Automate_cleaning.fetch_raw_data manual__2024-06-30T09:33:33.000476+00:00 [running]> on host 7ad299a7fc17
[2024-06-30T09:34:15.877+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=devi_nirfana
AIRFLOW_CTX_DAG_ID=Automate_cleaning
AIRFLOW_CTX_TASK_ID=fetch_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2024-06-30T09:33:33.000476+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-06-30T09:33:33.000476+00:00
[2024-06-30T09:34:41.550+0000] {local_task_job.py:156} INFO - Task exited with return code Negsignal.SIGKILL
[2024-06-30T09:34:41.746+0000] {taskinstance.py:1412} INFO - Marking task as FAILED. dag_id=Automate_cleaning, task_id=fetch_raw_data, execution_date=20240630T093333, start_date=20240630T093414, end_date=20240630T093441
[2024-06-30T09:34:42.016+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
